auto_tuning/proxy/src/proxy-runners/simple-runner.py

```diff
@@ -0,0 +1,21 @@
+import os, sys, inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
+
+import argparse
+import proxy_bindings as pb
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--cells', default=100000, type=int, help="num cells in a time cluster")
+parser.add_argument('-t', '--timesteps', default=20, type=int, help="num time steps/repeats")
+parser.add_argument('-k', '--kernel', default='all', type=str, help="kernel types")
+args = parser.parse_args()
```

Comment:
Here you can also use `choices` to validate that the user is using the correct kernels.

Reply:
done